packages/syft/src/syft/types/syft_object.py

```diff
@@ -4,16 +4,19 @@
 import inspect
 from inspect import Signature
 import types
+from typing import AbstractSet
 from typing import Any
 from typing import Callable
 from typing import ClassVar
 from typing import Dict
 from typing import KeysView
 from typing import List
+from typing import Mapping
```

Comment:
`typing.Mapping` is [deprecated](https://docs.python.org/3/library/typing.html#typing.Mapping) in favor of `collections.abc.Mapping` as well, but I think we use `typing.Mapping` a lot in our codebase we might want to do 1 separate PR that replaces all of them.